Character rotation cleanup and fixing

SAMPROO opened the pull request:
- Rotation done on one line
- Rotation bug where character was rotating on x and z axis FIXED
- Mouse rotation checked using UnityEngine.Plane and Plane.Raycast
- Mouse rotation to its own function
- Removed old dash function

SAMPROO committed:
Avatar tag

SAMPROO committed:
Merge branch 'master' into LocalController

SAMPROO committed:
LocalPlayerController rotate with one line

- rotate amount variable
- rotation only for y axis (bug fixed)
- rotate with gamepad or joystick separated to its own function

- using iiros grayboxed map

SAMPROO committed:
LocalPlayerController rotate with one line

- rotate amount variable
- rotation only for y axis (bug fixed)
- rotate with gamepad or joystick separated to its own function

SAMPROO committed:
Update sampo_character_builder.unity

- using iiros grayboxed map

SAMPROO committed:
Removed unecesarry rotation type function and created GetMouseRotation()

SAMPROO committed:
Merge branch 'LocalController' of https://github.com/Shophorn/morko into LocalController

SAMPROO committed:
Revert "LocalPlayerController rotate with one line"

This reverts commit 1892c3f0c3bc350c0bd8dd38f2d0fea64a3ad27f.

SAMPROO committed:
Revert "Revert "LocalPlayerController rotate with one line""

This reverts commit c1f35f4a6735ca92ff23ba60501cac2dbdc61eae.

Iiropel commented on MorkoUnityProject/Assets/Code/LocalPlayerController.cs:
```diff
@@ -245,109 +244,69 @@ private void Move(Vector3 moveDirection, bool accelerateRun)
-		currentMovementSpeed = 0f;
+
+		return mouseRotation;
 	}
```
funktion nimi ei liity mitenkään siihen mitä siinä tehdään:D 
(in english by google translate: the function name has nothing to do with what it does: D)
pitäis nykysellään olla RotationToCursorPositionRelativeToCameraPositionAndPlayerPosition() jos haluis et se vastais sitä mitä se tekee

Iiropel commented on MorkoUnityProject/Assets/Code/LocalPlayerController.cs:
```diff
@@ -2,6 +2,7 @@
 using System.Numerics;
 using UnityEngine;
 using Morko.Network;
+using Plane = UnityEngine.Plane;
```
Miks tää oli täällä

SAMPROO replied:
Koska LocalPlayerController ei implementoi MonoBehaviouria

Iiropel reviewed (commented):
👍

SAMPROO committed:
Changed mouse rotation function naming

SAMPROO committed:
Updated cursor rotation function naming

SAMPROO committed:
Merge branch 'LocalController' of https://github.com/Shophorn/morko into LocalController

SAMPROO committed:
Created tester character prefab for ProtoGameCreator

- added character creation for testing purposes to ProtoGameCreator
- Created Character collection for testing purposes
